bitnami/harbor/templates/clair/clair-dpl.yaml

```diff
@@ -0,0 +1,89 @@
+{{ if .Values.clair.enabled }}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: "{{ template "harbor.clair" . }}"
+  labels:
+    {{ include "harbor.labels" . | nindent 4 }}
+    app.kubernetes.io/component: clair
+spec:
+  replicas: {{ .Values.clair.replicas }}
+  selector:
+    matchLabels:
+      {{ include "harbor.matchLabels" . | nindent 6 }}
+      app.kubernetes.io/component: clair
+  template:
+    metadata:
+      labels:
+        {{ include "harbor.labels" . | nindent 8 }}
+        app.kubernetes.io/component: clair
+      annotations:
+        checksum/configmap: {{ include (print $.Template.BasePath "/clair/clair-cm.yaml") . | sha256sum }}
+      {{- if .Values.clair.podAnnotations }}
+        {{ toYaml .Values.clair.podAnnotations | nindent 8 }}
+      {{- end }}
+    spec:
+      {{- with .Values.imagePullSecrets }}
+      imagePullSecrets:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      containers:
+      - name: clair
+        image:  "{{ template "harbor.clairImage" . }}"
+        imagePullPolicy: {{ .Values.clairImage.pullPolicy | quote }}
+        {{- if .Values.clair.livenessProbe.enabled }}
+        livenessProbe:
+          httpGet:
+            path: /health
+            port: 6061
+          initialDelaySeconds: 30
+          periodSeconds: 10
+        {{- end }}
+        {{- if .Values.clair.readinessProbe.enabled }}
+        readinessProbe:
+          httpGet:
+            path: /health
+            port: 6061
+          initialDelaySeconds: 30
+          periodSeconds: 10
+        {{- end }}
+        env:
+        - name: BITNAMI_DEBUG
+          value: {{ ternary "true" "false" .Values.registryImage.debug | quote }}
+        {{- if .Values.clair.httpProxy }}
+        - name: HTTP_PROXY
+          value: {{ .Values.clair.httpProxy }}
+        {{- end }}
+        {{- if .Values.clair.httpsProxy }}
+        - name: HTTPS_PROXY
+          value: {{ .Values.clair.httpsProxy }}
+        {{- end }}
+        - name: NO_PROXY
+          value: "{{ template "harbor.registry" . }},{{ template "harbor.core" . }}"
+        {{- if .Values.clair.resources }}
+        resources:
+          {{ toYaml .Values.clair.resources | nindent 10 }}
+        {{- end }}
+        ports:
+        - containerPort: 6060
+        volumeMounts:
+        - name: clair-config
+          mountPath: /etc/clair/config.yaml
+          subPath: config.yaml
+      volumes:
+      - name: clair-config
+        configMap:
+          name: "{{ template "harbor.clair" . }}"
+          items:
+            - key: config.yaml
+              path: config.yaml
```

Comment:
Be consistent with indentation. In the rest of the file you're describing arrays using:
```
array:
- a
- b
- ...
```
Here, you're using:
```
array:
  - a
  - b
  - ...
```

Reply:
done